launch-or-update-azure.sh

```diff
@@ -188,8 +188,9 @@ if [ -z ${namespace+x} ]
 then 
   namespace=`cat /dev/urandom | tr -dc 'a-zA-Z' | fold -w 8 | head -n 1`
   echo "namespace not set: generated $namespace"
-else #truncate to 63 chars
+else #truncate to 63 chars && convert to lowercase to be DNS compliant
   namespace=`echo "$namespace" | cut -c 1-63`
```

Comment:
we should probably be consistent on ` vs $(
Can you please move them all to $(, as I believe that's better?